### PR TITLE
sci-astronomy/wcslib: add 7.12

### PR DESCRIPTION
**Pull request by stkw0**

### CODE REVIEW

File: sci-astronomy/wcslib/wcslib-7.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	fi
+	econf "${myconf[@]}"
+	sed -i -e 's/COPYING\*//' GNUmakefile || die
+}
+
+src_install () {
+	default
+	# static libs share the same symbols as shared (i.e. compiled with PIC)
+	# so they are not compiled twice
+	use static-libs || rm "${ED}"/usr/$(get_libdir)/lib*.a
```
Comment (AndrewAmmerlaan) [inline sci-astronomy/wcslib/wcslib-7.12.ebuild:62]:
```
if ! use static-libs; then
    rm "${ED}"/usr/$(get_libdir)/lib*.a || die
fi
```
We need to add a `|| die` to ensure the ebuild properly dies if rm fails.

File: sci-astronomy/wcslib/wcslib-7.12.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	use doc || rm -r \
+		"${ED}"/usr/share/doc/${PF}/html \
+		"${ED}"/usr/share/doc/${PF}/*.pdf
```
Comment (AndrewAmmerlaan) [inline sci-astronomy/wcslib/wcslib-7.12.ebuild:65]: Similarly here

**Review by AndrewAmmerlaan — COMMENTED** [review]
Looks good :+1: 

Could you please add a `Signed-off-by` line to your commit message, in accordance with Gentoo's [copyright policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin)?

**Comment by stkw0** [comment]
Sorry, I used a computer in which I don't have the Signed-off-by configured

**Comment by AndrewAmmerlaan** [comment]
Nice work, Thanks :+1:

**Pull request closed without merging**